pkg/cluster/cluster_info_scraper.go

```diff
@@ -249,6 +246,19 @@ func (s *ClusterScraper) findRunningPodsOnNode(nodeName string) ([]*api.Pod, err
 	return s.GetPods(api.NamespaceAll, listOption)
 }
```

Comment:
Looks this `findRunningPodsOnNode` func is not used any more.

Reply:
Fix by removing this function.